Formula/hg-fast-export.rb

```diff
@@ -0,0 +1,19 @@
+class HgFastExport < Formula
+  desc "Mercurial to git converter using git-fast-import"
+  homepage "https://packages.debian.org/source/sid/hg-fast-export"
+  url "http://http.debian.net/debian/pool/main/h/hg-fast-export/hg-fast-export_20140308.orig.tar.gz"
```

Comment:
If we're using Debian we should be using one of the secure mirrors as well. No reason to ever use plaintext with Debian really. https://mirrors.ocf.berkeley.edu/debian/pool/main/ or mirrorservice are the standard choices.